README.md

```diff
@@ -33,30 +33,60 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
-   MacOS:
+   **MacOS**:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
    ```
 
-   Ubuntu (18.04LTS or later):
+   You can also download a pre-built RISC-V toolchain
+   [here](http://cs.virginia.edu/~bjc8c/archive/gcc-riscv64-unknown-elf-8.3.0-mac.zip).
+
+   **Ubuntu (18.04LTS or later)**:
    ```
    $ sudo apt install gcc-arm-none-eabi
+   $ sudo apt install gcc-riscv64-unknown-elf
+   ```
+
+   Unfortunately, Ubuntu does not currently (June 2020) provide a package for
```

Comment:
It's probably best to specify the version as earlier then 20.10

Reply:
What happens if this isn't fixed in 20.10?

Reply:
Then we will have to update it, but a date has the same problem. What if 20.10 works? It's then not clear to people that it should work for them.

Either way though we will need to update the steps.